src/handlers/selecting.handler.js

```diff
@@ -0,0 +1,43 @@
+import { questionView } from '../views/question.view.js';
+import { quizData } from '../data.js';
+
+
+export const selectingHandler = (event) => {
+
+// const newArrays = Object.values(quizData.questions[i].answers)
+// const quesAnswersKeys = Object.keys(quizData.questions[i].answers)
+// console.log(newArrays)
+// console.log(quesAnswersKeys)
+
+let i = 1; // i will be currentQuestion
```

Comment:
Remove this variable, as it is no longer used.

Reply:
Yes ,
I removed it